ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+---
+arc: 33
+title: Becoming an xGov
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
+ 
+The purpose of this ARC is to define the standard by which xGov status is achieved.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
+xGov **MAY** participate in the designation of proposals.
+ 
+### Requirement to become an xGov
+The Foundation suggests that to become an xGov an Algo holder **MUST**:
```

Comment:
This is not super clear to me. Do we need to participate in another reward-bearing program on top of the regular governance or is regular governance enough to qualify? Can we become xGov without being regular governors but participating in another qualified reward-bearing AF program?

Reply:
from their previous posts , I think they're referring to normal Algorand governance, but the fact they use the word "another" doesn't make sense to me

Reply:
Anybody thought of making this not 1:1 or 1 per "person" but a quadratic voting + KYC/SBT to prevent Sybil attacks ?
This would increase the chances of out voting the whales but still give higher weights to larger stake ?

Reply:
I agree with this idea

We could use decentralized identity so no one can use multiple wallets. That way people cannot cheat. 

For quadratic voting, perhaps it can be applied if the balance you have in your wallet is above a certain amount ? If we do decentralized identity, then no one would be able to cheat by disbursing their amounts to different wallets.